src/scripts/install.sh

```diff
@@ -1,23 +1,27 @@
-if echo $OSTYPE | grep darwin > /dev/null 2>&1; then
+if echo "$OSTYPE "| grep darwin > /dev/null 2>&1; then
     brew install shellcheck
     exit $?
 fi
 
-if cat /etc/issue | grep Alpine > /dev/null 2>&1; then
+if grep Alpine /etc/issue > /dev/null 2>&1; then
     apk add shellcheck
     exit $?
 fi
 
-if cat /etc/issue | grep Debian > /dev/null 2>&1 || cat /etc/issue | grep Ubuntu > /dev/null 2>&1; then
+if  grep Debian /etc/issue > /dev/null 2>&1 || grep Ubuntu /etc/issue > /dev/null 2>&1; then
     if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we're root
         export SUDO="sudo";
     fi
-    $SUDO apt update
-    $SUDO apt install -y shellcheck
+    # This is currently installing an outdated version of Shellcheck.
+    # $SUDO apt update
+    # $SUDO apt install -y shellcheck
+    wget -qO- https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz | tar -xJf -
```

Comment:
If we take this approach (immutable orbs have a set version of a binary), we'll have to approach maintenance in the same way we do with Docker images. I think it's "safer" to allow a custom version to be defined, and default to a "known good" version if we want to avoid always installing latest. Otherwise we'll have to manually update the version and post a new semver for every release.

Reply:
^ This was only done for testing as the APT version of shellcheck is out of date. In order to get all OS's on the same version of shellcheck, we would have to manually install like this. Otherwise we can only support up to the version on apt.

Reply:
If we set a version tag, we'll need to use the releases like this for the other OS too